v2/src/domain.ts

```diff
@@ -3,7 +3,7 @@ export interface Saksbehandler {
     readonly fornavn: string;
     readonly etternavn: string;
     readonly navn: string;
-    readonly enheter: Array<Enhet>;
+    readonly enheter?: Array<Enhet>;
```

Comment:
Problemet er at hele kallet for hente ut `Saksbehandler` feiler, så trenger ikke denne endringen.